### PR TITLE
You need to delete all testing comment at app.js

### PR DESCRIPTION
**Pull request by Pugaden**
Just delete all comments in app.js

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -101,9 +101,3 @@ app.use(globalErrorHandler)
 
 module.exports = app
 
-
-// test issue #2
-// this coment fixes issue #2
```
Comment (DenysPugachov) [inline app.js:106]: this is a comment

**Review by DenysPugachov — COMMENTED** [review]
Check the coment 108

File: app.js (hunk 1 above)
Comment (Pugaden) [reply app.js:106]: this is an answer form PugaDen

Comment (DenysPugachov) [reply app.js:106]: ok, let's do it!

Comment (Pugaden) [reply app.js:106]: ok.

**Comment by DenysPugachov** [comment]
Deleting all comment in app.js like PugaDen suggested.

**Pull request merged**